src/Evolution/DgSubcell/ActiveGrid.cpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+
+#include <ostream>
+
+#include "Utilities/ErrorHandling/Error.hpp"
+
+namespace evolution::dg::subcell {
+std::ostream& operator<<(std::ostream& os, ActiveGrid t) noexcept {
```

Comment:
[very minor, feel free to veto] I suggest `t` -> `active_grid` for slight clarity improvement. If you make this change, also do it in the header file.